core/common/src/main/java/alluxio/resource/DynamicResourcePool.java

```diff
@@ -181,26 +178,15 @@ public static Options defaultOptions() {
   private final int mMaxCapacity;
   private final int mMinCapacity;
 
-  // Tracks the resources that are available ordered by lastAccessTime (the first one is
+  // Tracks the resources that are available ordered by lastAccessTime (the head is
```

Comment:
explicitly say:
- these are the resources that acquire() will take
- this is always a subset of the other data structure `mResources`

NIT: I prefer a plural name, like `mFreeResources` or `mAvailableResources`.